[2025-08-18 09:15:49,377] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline manual__2025-08-18T09:15:47.545529+00:00 [queued]>
[2025-08-18 09:15:49,385] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline manual__2025-08-18T09:15:47.545529+00:00 [queued]>
[2025-08-18 09:15:49,385] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 09:15:49,385] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-18 09:15:49,385] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 09:15:49,395] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): verify_pipeline> on 2025-08-18 09:15:47.545529+00:00
[2025-08-18 09:15:49,397] {standard_task_runner.py:52} INFO - Started process 508 to run task
[2025-08-18 09:15:49,400] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'verify_pipeline', 'manual__2025-08-18T09:15:47.545529+00:00', '--job-id', '270', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmpp4moe8hf', '--error-file', '/tmp/tmp8v282f8d']
[2025-08-18 09:15:49,401] {standard_task_runner.py:80} INFO - Job 270: Subtask verify_pipeline
[2025-08-18 09:15:49,465] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.verify_pipeline manual__2025-08-18T09:15:47.545529+00:00 [running]> on host a4b544519f2b
[2025-08-18 09:15:49,552] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=verify_pipeline
AIRFLOW_CTX_EXECUTION_DATE=2025-08-18T09:15:47.545529+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-08-18T09:15:47.545529+00:00
[2025-08-18 09:15:49,553] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-18 09:15:49,554] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            echo " Attente traitement Logstash..." && sleep 30 &&\n            curl -s "http://elasticsearch:9200/_cat/indices/weather*" || echo "Pas encore d\'index weather"\n        ']
[2025-08-18 09:15:49,565] {subprocess.py:85} INFO - Output:
[2025-08-18 09:15:49,568] {subprocess.py:92} INFO -  Attente traitement Logstash...
[2025-08-18 09:16:19,605] {subprocess.py:92} INFO - yellow open weather_data BQjbDtS5TvO1Q2HrAJGyvA 1 1 65 0 70.5kb 70.5kb
[2025-08-18 09:16:19,607] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-18 09:16:19,646] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=verify_pipeline, execution_date=20250818T091547, start_date=20250818T091549, end_date=20250818T091619
[2025-08-18 09:16:19,673] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-18 09:16:19,706] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
